src/index.html

```diff
@@ -1,13 +1,368 @@
+
+            </div>
+          </div>
+          <div class="details__bicycle bicycle">
+            <div class="bicycle__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4 grid__item--tablet-1-2">
+                  <div class="bicycle__photo-container">
+                    <img
+                    src="../src/images/photos/5.jpg"
+                    alt="photo_5"
+                    class="bicycle__photo"
+                  >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--desktop-5-12
+                grid__item--tablet-3-6">
+                  <div class="bicycle__photo-container">
+                    <img
+                  src="../src/images/photos/6.jpg"
+                  alt="photo_6"
+                  class="bicycle__photo"
+                  >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="bicycle__description">
+              <h2 class="bicycle__name">Hydraulic disc brakes & Lightweight</h2>
+              <p class="bicycle__name-description">Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.</p>
+
+            </div>
+          </div>
+          <div class="grid grid--desktop">
+            <div class="grid__item
+            grid__item--2-5 grid__item--5-8">
+              <a href="#" class="button">Explore</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-5">
+              <form action="#">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contacts__field form-field"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contacts__field form-field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contacts__field--last
+                    contacts__field
+                    form-field
+                    form-field--textarea"
+                >
+                </textarea>
+
+                <button class="button button--primary" id="button">
+                  Send
+                </button>
+
+              </form>
+            </div>
+            <div class="grid__item grid__item--8-9">
```

Comment:
Checklist | point 4 - you should add some hover effects to change color of the links